File: found.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        string text = "The quick brown fox jumps over the lazy fox and sees another fox";
        string wordToFind = "fox";
        WordIndexFinder(wordToFind, text);
    }

    static void WordIndexFinder(string wordToFind, string text)
    {
        string[] words = text.Split(' ');
        List<int> indices = new List<int>();

        for (int index = 0; index < words.Length; index++)
        {
            if (words[index] == wordToFind)
            {
                indices.Add(index);
            }
        }

        if (indices.Count > 0)
        {
            Console.WriteLine($"The word '{wordToFind}' is found at indices: {string.Join(", ", indices)}");
        }
        else
        {
            Console.WriteLine($"The word '{wordToFind}' is not in the list.");
        }
    }
}

File: found.py
text = "The quick brown fox jumps over the lazy fox and sees another fox"
word_to_find = "fox"

def wordIndexFounder(word_to_find, text):
    words = text.split()
    indices = []
    for index, word in enumerate(words):
        if word == word_to_find:
            indices.append(index)
    if indices:
        print(f"The word '{word_to_find}' is found at indices: {indices}")
    else:
        print(f"The word '{word_to_find}' is not in the list.")

wordIndexFounder(word_to_find, text)